java-jacoco/src/main/java/org/sonar/plugins/jacoco/AbstractAnalyzer.java

```diff
@@ -273,6 +264,7 @@ private static void analyzeFile(NewCoverage newCoverage, InputFile resource, ISo
 
   protected abstract CoverageType coverageType();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "CoverageType"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)